### ISSUE TITLE
Blank App in Teams

### ISSUE THREAD

**Issue by CSBenW** [OfficeDev/microsoft-teams-emergency-operations-center#140]
We are having an issue with the Teams App not displaying any content after deployment. The title bar is there with the info, help, and feedback icons. The "welcome" message is also displayed, but there is no content below that to actually use the app. I did confirm the REACT_APP_SHAREPOINT_SITE_NAME is set properly in Azure. I have also verified the issue occurs in the browser version of Teams, as well as the desktop version. I will try to get the console logs to share for further troubleshooting. I was able to get the console log exported. Is there a secure way for me to share that with you here?
![256834757-3c2d71bc-269b-4ee2-9b81-104431c4b8b9](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/assets/133781988/8b66b18e-1282-416d-b6d6-b4e3f3c89041)

**Comment by v-asshrirao** [comment]
@CSBenW, Thank you for reaching out to us. You can attach the log file here to help us troubleshoot the issue.

**Comment by CSBenW** [comment]
[teams.microsoft.com-1694091952860.log](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/files/12730291/teams.microsoft.com-1694091952860.log)

**Comment by v-asshrirao** [comment]
@CSBenW , Thank you for sharing logs with us.
Can you please check the REACT_APP_SHAREPOINT_SITE_NAME value from AppService. It should be only site name i.e. TEOCTestSite. 

Once updated perform this [step](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/wiki/Upgrade-to-Latest-Version#3-sync-latest-version), to get the code updated with the new site name value in your app service. Once that is done, open the app in teams and verify if that works as expected.

**Comment by bderry71** [comment]
I'm getting the same thing. I checked the REACT_APP_SHAREPOINT_SITE_NAME and it is set to the SharePoint Site I built for this app. I need an overview of the architecture of this system. Can it be added to any Team and it lives inside of Azure with the SharePoint Site as database? I've deployed it to a team then had it working once. I came back to rebuild it and deleted the team. I rebuilt it with the same name. Then it won't work anymore. Then I did it into a new team with a different name and it still is not launching the dashboard. Blank screen like OP above.

**Comment by v-asshrirao** [comment]
@bderry71 , Please open new issue and share console logs and network trace by opening the TEOC app in browser version of teams.

**Comment by v-asshrirao** [comment]
@CSBenW, Please confirm if you are still facing the issue so we can close this.

**Comment by v-asshrirao** [comment]
@CSBenW , Can you please share screenshot with us for validation.
![image](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/assets/141898119/19fb7ce7-99a5-4528-b42c-c43d96e84ade)

Also request you to perform this [step](https://github.com/OfficeDev/microsoft-teams-emergency-operations-center/wiki/Upgrade-to-Latest-Version#3-sync-latest-version) and share logs and network trace with us along with manifest files for troubleshooting.
